estuary_updater/handlers/freshmaker.py

```diff
@@ -79,13 +80,57 @@ def build_state_handler(self, msg):
         """
         build_info = msg['body']['msg']
         event_id = msg['body']['msg']['event_id']
+        freshmaker_build = self.create_or_update_freshmaker_build(build_info, event_id)
         build = self.create_or_update_build(build_info, event_id)
-        if build:
-            event = FreshmakerEvent.nodes.get_or_none(id_=str(event_id))
-            if event:
-                event.triggered_container_builds.connect(build)
-            else:
-                log.warn('The Freshmaker event {0} does not exist in Neo4j'.format(event_id))
+        event = FreshmakerEvent.nodes.get_or_none(id_=str(event_id))
+        if event:
+            if build:
+                event.successful_koji_builds.connect(build)
+            if freshmaker_build:
+                freshmaker_build.event.connect(event)
```

Comment:
After https://github.com/release-engineering/estuary-api/pull/238, this would need to be `conditional_connect` (method on the `EstuaryStructuredNode` class) to avoid potential cardinality violations. It's safe to set it here now even before the PR I mentioned is merged.